tests/backend/checkmach/fail3.output

```diff
@@ -0,0 +1,2 @@
+File "fail3.ml", line 1:
```

Comment:
(That line number is suspicious.
Since the information is probably
long gone when we output, we
should probably strip the line
number or use a nonsensical value).